packages/kbn-test/src/functional_tests/lib/fips_overrides.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// This will only apply overrides when running in FIPS mode
+export function applyFipsOverrides(vars: any) {
+  if (process.env.FTR_ENABLE_FIPS_AGENT?.toLowerCase() === 'true') {
+    vars.esTestCluster.license = 'trial';
+
+    const skipTags = vars.suiteTags?.exclude ?? [];
+    skipTags.push('skipFIPS');
+    vars.suiteTags = {
+      ...vars.suiteTags,
+      exclude: skipTags,
+    };
+
+    vars.security = {
+      ...vars.security,
+      defaultRoles: ['superuser', 'kibana_admin', 'system_indices_superuser'],
```

Comment:
>Kibana requires security to be enabled and a platinum or better license to run in FIPS mode

_question_ : does this change is targeting only stateful self-managed clusters? It would be helpful to have a comment explicitly describing the cases and also explain the roles selection

Reply:
++I will add a comment to clarify what is happening and why

Currently we are only targeting stateful for FIPS: Phase 1, but I wanted to run all tests possible, so these overrides are applying on serverless tests _when run with the FIPS CI label_